sdk/compute/mgmt/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineExtensionImagesImpl.java

```diff
@@ -44,9 +45,10 @@ public VirtualMachineExtensionImagesImpl(VirtualMachinePublishers publishers) {
                     virtualMachinePublisher
                         .extensionTypes()
                         .listAsync()
-                        .onErrorResume(e -> Mono.empty())
-                        .flatMap(
-                            virtualMachineExtensionImageType -> virtualMachineExtensionImageType.versions().listAsync())
+                        .onErrorResume(ManagementException.class,
+                            e -> e.getResponse().getStatusCode() == 404 ? Flux.empty() : Flux.error(e))
```

Comment:
I think it could just be removed, due to other list will not deal with 404.

Reply:
This is because for 1 or 2 publisher, above list image type would fail with 404 (error on publisher not valid). But apparently we would still want to continue with the rest publishers.